WooCommerce/Classes/ViewRelated/Products/Edit Product/Cells/ImageAndTitleAndTextTableViewCell.swift

```diff
@@ -132,6 +132,7 @@ final class ImageAndTitleAndTextTableViewCell: UITableViewCell {
         configureContentStackView()
         configureTitleAndTextStackView()
         applyDefaultBackgroundStyle()
+        configureBackground()
```

Comment:
A suggestion: How about renaming this method to `configureSelectedBackground`? Since this method takes care of the selected background, and we already handled the default background on the line above so I think we can be more specific here 😄 

We can even move the configuration to the `UITableViewCell` extension like [what we did](https://github.com/woocommerce/woocommerce-ios/blob/e8ae4c31a5d6e4d6235d62e9bb528fe6186dadd6/WooCommerce/Classes/Extensions/UITableViewCell%2BHelpers.swift#L17-L19) with `applyDefaultBackgroundStyle`. The point is to reuse this piece of code on different table cells without having to duplicate it. WDYT?